definitions/npm/redux-saga_v0.11.x/flow_>=v0.28.x/redux-saga_v0.11.x.js

```diff
@@ -87,38 +87,35 @@ declare module 'redux-saga' {
   declare type Saga5<Y: IOEffect, R, N, T1, T2, T3, T4, T5> = (t1: T1, t2: T2, t3: T3, t4: T4, t5: T5) => Generator<Y, R, N>;
   declare type Saga6<Y: IOEffect, R, N, T1, T2, T3, T4, T5, T6> = (t1: T1, t2: T2, t3: T3, t4: T4, t5: T5, t6: T6) => Generator<Y, R, N>;
 
-  declare type TakeXRet = {
-    @@iterator(): $Iterator<*,*,*>,
+  declare interface TakeXRet extends Generator<*,*,*> {
```

Comment:
💭 I wasn't sure if this was idiomatic or if `Generator<*,*,*> & { name: string }` was more-so. They seem to be the same, yeah?